manifests/v1alpha2/vizier/suggestion/grid/deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: extensions/v1beta1
```

Comment:
Is "vizier" necessary here? I think we can just use "suggestion" here. Also going forward we should avoid using the name vizier since it can confuse users.